environment.yml

```diff
@@ -4,15 +4,23 @@ channels:
   - mantid/label/nightly
   - oncat
```

Comment:
pyoncat is not being installed in the environment and is not used. Can this be removed?